.github/workflows/publish-extension-nightly.yml

```diff
@@ -90,6 +90,10 @@ jobs:
         run: |
           yarn build:patch:local:ci:nightly ${{ env.PUBLISHING_TARGET }}
 
+      - name: Build the VSIX w/o Target
```

Comment:
We need to run this as a separate matrix entry, else we'll build this 6 times redundantly - and also in the next 'Check for VSIX' step, we'll have two vsix's instead of one.  Some Tweaks:

```yaml
      matrix:
        include:
          - platform: win32
            arch: x64
          - platform: win32
            arch: ia32
          - platform: linux
            arch: x64
          - platform: linux
            arch: arm64
          - platform: darwin
            arch: x64
          - platform: darwin
            arch: arm64
          - platform: general
            arch: general
```

```yaml
      - name: Download SQLite Binary
        run: |
          if [ ${{ matrix.platform }} == linux ]; then
            yarn download-sqlite-binary --target_arch=${{ matrix.arch }}
          elif [ ${{ matrix.platform }} != general ]; then
            yarn download-sqlite-binary --target_arch=${{ matrix.arch }} --target_platform=${{ matrix.platform }} --target_libc=unknown
          else 
            echo "Skipping SQLite binary install for general build."
          fi
```

```yaml
- name: Build the VSIX
run: |
  if [] ${{ matrix.platform }} == general ; then
     yarn build:patch:local:ci:nightly:noparam
  else 
    yarn build:patch:local:ci:nightly ${{ env.PUBLISHING_TARGET }}
  fi
```

Might need to check syntax/typos in this block, I didn't test it.